data2bids.m

```diff
@@ -633,9 +633,11 @@
 cfg.channels.wavelength_actual          = ft_getopt(cfg.channels, 'wavelength_actual'           , nan);
 cfg.channels.wavelength_emission_actual = ft_getopt(cfg.channels, 'wavelength_emission_actual'  , nan);
 cfg.channels.short_channel              = ft_getopt(cfg.channels, 'short_channel'               , nan);
-
 % specific options for motion channels
-% cfg.channels.tracked_point         = ft_getopt(cfg.channels, 'tracked_point'  , nan);
+cfg.channels.sampling_frequency         = ft_getopt(cfg.channels, 'sampling_frequency'          , nan);
+cfg.channels.component                  = ft_getopt(cfg.channels, 'component'                   , nan);
+cfg.channels.tracked_point              = ft_getopt(cfg.channels, 'tracked_point'               , nan);
+
```

Comment:
I would propose adding "tracking_system" as this is a required column specified in the google doc.

cfg.channels.tracking_system              = ft_getopt(cfg.channels, 'tracking_system'               , nan);